Configuration/Geometry/README.md

```diff
@@ -46,5 +48,7 @@ Several detector combinations have been generated:
 * D48 = T16+C9+M3+I10+O3+F2
 * D49 = T15+C9+M4+I10+O4+F2
 * D50 = T15+C9+M4+I11+O4+F2
+* D51 = T17+C9+M4+I10+O4+F2
```

Comment:
It was not clear to me whether I should use M3 or M4, O3 or O4, I10 or I11.
Please let me know in case change is needed here :)

Reply:
I think you made the right choice to start from D49, since that is the new baseline for the HLT TDR. (M4 and O4 should always go together.)

Reply:
thanks @kpedro88

Reply:
@ghugo83 @kpedro88 O11 is equivalent to O10 as physics content, it is mostly a validation/migration tool for me, D49 is a meaningful base